service/controller/key/key.go

```diff
@@ -52,7 +53,21 @@ const (
 	CertificateEncryptionIVName    = "encryptioniv"
 
 	CoreosVersion = "2303.4.0"
+)
+
+// Container image versions for k8scloudconfig.
+const (
+	// kubectl container version.
+	kubectlVersion = "1.16.4"
+
+	// k8s-api-healthz version.
+	kubernetesAPIHealthzVersion = "0999549a4c334b646288d08bd2c781c6aae2e12f"
+
+	// k8s-setup-network-environment.
+	kubernetesNetworkSetupDocker = "1f4ffc52095ac368847ce3428ea99b257003d9b9"
```

Comment:
@giantswarm/team-celestial I just noticed that this is a very old image corresponding to https://github.com/giantswarm/k8s-setup-network-environment/pull/2 I think. The latest is `68e90113331feca3b9ffe6a75a601b381ba8c1f7` (from https://quay.io/repository/giantswarm/k8s-setup-network-environment?tab=tags). I'm not sure the implications of this regression--just wanted to mention.